CS.Security/Interfaces/ITokenGenerator.cs

```diff
@@ -1,16 +1,11 @@
 using CS.Security.DTO;
 using CS.Security.Models;
 
-using CS.Security.Services.Authentication;
+namespace CS.Security.Interfaces;
 
-namespace CS.Security.Interfaces
+public interface ITokenGenerator
 {
-    public interface ITokenGenerator
-    {
-        Task<string> GenerateAccessToken(User user);
-        string GenerateRefreshToken(User user);
-        Task<TokenDto> GenerateTokens(User user);
-        Task<TokenDto> RefreshAccessToken(string accessToken, string refreshToken);
-
-    }
-}
+    Task<TokenDto> GenerateTokens(User user);
```

Comment:
there were two methods used only within the TokenGenerator code, not exposed outside, so they should be made private
again, ReSharper/Rider helps to find issues like that